openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -1110,6 +1113,7 @@ The following parameters are used in the JSON-encoded Credential Response body:
    * `credential`: REQUIRED. Contains one issued Credential. It MAY be a string or an object, depending on the Credential Format. See Appendix A for the Credential Format-specific encoding requirements.
 * `transaction_id`: OPTIONAL. String identifying a Deferred Issuance transaction. This parameter is contained in the response if the Credential Issuer cannot immediately issue the Credential. The value is subsequently used to obtain the respective Credential with the Deferred Credential Endpoint (see (#deferred-credential-issuance)). It MUST not be used if the `credentials` parameter is present. It MUST be invalidated after the Credential for which it was meant has been obtained by the Wallet.
 * `notification_id`: OPTIONAL. String identifying one or more Credentials issued in one Credential Response. It MUST be included in the Notification Request as defined in (#notification). It MUST not be used if the `credentials` parameter is not present.
+* `credential_dataset_id`: OPTIONAL.  An opaque string containing the Credential Dataset Identifier associated with the returned Credential(s). This allows Wallets to detect changes to the underlying Credential Dataset across different Credential Responses.
```

Comment:
Personally I think this feature is important enough that this identifier should be mandatory to return otherwise it makes it very difficult for credential update/refresh to be supported. If its left optional we need to discuss how it gets signalled whether this feature is supported and what a wallet is suppose to assume when it comes to updating/refreshing credentials.